08-regression2.Rmd

```diff
@@ -240,7 +242,7 @@ regression, we can start to compare and contrast these methods as well as the
 predictions made by them. To start, let's look at the visualization of the
 simple linear regression model predictions for the Sacramento real estate data
 (predicting price from house size) and the "best" K-NN regression model
-obtained from the same problem:
+obtained from the same problem, shown in Figure \@ref(fig:08-compareRegression).
 
 ```{r 08-compareRegression, echo = FALSE, warning = FALSE, message = FALSE, fig.height = 4, fig.width = 10, fig.cap = "Comparison of simple linear regression and K-NN regression"}
 set.seed(1234)
```

Comment:
The line below with the code `add_recipe(sacr_recipe)` is causing an error when I try to execute the chapter independently because it doesn't show up anywhere else in the chapter (`sacr_recipe`) is not defined.